.github/workflows/build.yml

```diff
@@ -1,15 +1,17 @@
-name: CMake Build
+name: Build
 on: [push, pull_request]
 
 jobs:
   linux:
     name: ${{ matrix.config.name }}
     runs-on: ubuntu-latest
+    container: ubuntu:20.04
     env:
       inexor_build_type: "Release"
       inexor_conan_path: "$HOME/.local/bin"
       inexor_vulkan_sdk: "$GITHUB_WORKSPACE/vulkan_sdk/"
       inexor_vulkan_version: "1.2.131.1"
+      DEBIAN_FRONTEND: "noninteractive"
```

Comment:
Bit nit picky but, maybe `DEBIAN_FRONTEND` should go at the top of `env` to be in alphabetical order :).